tests/platform/test_platform.py

```diff
@@ -149,10 +149,10 @@ def get_anneal_qprogram(runcard, flux_to_bus_topology):  # pylint: disable=too-m
     platform.flux_to_bus_topology = flux_to_bus_topology
     anneal_waveforms = {
         next(element.bus for element in platform.flux_to_bus_topology if element.flux == "phix_q0"): Arbitrary(
-            np.array([1])
+            np.array([0.0, 0.0, 0.0, 1.0])
         ),
         next(element.bus for element in platform.flux_to_bus_topology if element.flux == "phiz_q0"): Arbitrary(
-            np.array([2])
+            np.array([0.0, 0.0, 0.0, 2.0])
```

Comment:
Can we test more complex arbitray shapes just in case? If it makes sense of course

Reply:
i dont think it makes a lot of sense at this step, since we already have somewhat more complicated waveforms at the `get_waveform`test. Even those are not quite complicated, but making more complex arrays won't make a difference since we're not operating with them that much (i..e. there's not gonna be any difference between a size 3 array with 2 0s and 1 random float and a 10k long array with floats, and the later would make reviewing/updating the tests more difficult in the future)